### PR TITLE
[15.0][FIX] analytic operating unit: domain ou in users

### PR DESCRIPTION
**Pull request by Saran440**
[BUG]
user (multi ou) can't see other ou in analytic

This pr fix domain in analytic can select other ou in analytic

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Thanks for the fix. Can you put the module name in the commit message? just for better reading when looking at the history.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
Incomplete fix. The complete one is https://github.com/OCA/operating-unit/pull/400. Could you cherry-pick it instead or complete your own? Also a Pr for v13 and v14 should be needed, right?

**Comment by Saran440** [comment]
@MiquelRForgeFlow I tested with version15 it's not working.
and why we have this domain? I think user can see ou following access right.

Peek 2023-01-26 18-22

File: analytic_operating_unit/views/analytic_account_view.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
                 <field
                     name="operating_unit_ids"
                     widget="many2many_tags"
+                    domain="[('user_ids', '=', uid)]"
```
Comment (MiquelRForgeFlow) [inline analytic_operating_unit/views/analytic_account_view.xml]: Oh well, if the module doesn't work with this line, then remove it :S I just thought that it was needed as it was solved this way in v12.

Comment (AaronHForgeFlow) [reply analytic_operating_unit/views/analytic_account_view.xml]: I confirm that fix do not work in v15. Can you remove that part?

**Comment by Saran440** [comment]
@MiquelRForgeFlow @AaronHForgeFlow I fixed it by cherry-pick https://github.com/OCA/operating-unit/pull/400 and remove domain in xml file

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**